docs/api/recordings.md

```diff
@@ -138,3 +139,18 @@ Content-Type: application/json
   "userId": "24"
 }
 ```
+
+
+# Recording result set
+
+In order to group recordings together a result set is used. The result set
+contains multiple recordings and an audio file containing all the audio files
+of its recordings.
+
+A result set is created automatically whenever a recording is created with
+a non-existing `resultSetId`. When the `resultSetId` does exist the recording
+is added to this set and the audio of the recording is appended to the audio
+of the result set.
+
+The result set cannot be queried directly but it can be seen when getting
+[progress](progress.md).
```

Comment:
This implies that I would have to use the progress api to playback audio for the student. That is not what currently happens.

I was under the impression that the GET for a SpeechRecording would also return the the collection (=set) right?
So it would return either the the recording (just as it does now) or it returns the set of recordings?

Reply:
The recordings api returns a recording (without the result set). The result set id should be in there (I forgot to add that).
This is why I opted for a separate result set api.

Reply:
:thinking: how? Embedding the result set partially or adding the param to the recording?

Reply:
Something like this:

### Request

 ```http
 GET /challenges/speech/speech_1/recordings/recording_1 HTTP/1.1
 Accept: application/json
 ```
 
 ### Response
 
 ```http
 HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
   "id": "speech_1",
   "created": "2014-01-28T21:25:10Z",
   "updated": "2014-01-28T21:25:10Z",
   "audioUrl": "https://api.itslanguage.nl/download/YsjdG37bUGseu8-bsJ",
   "userId": "24",
   "resultSetId": "4everAndever",
   "combinedAudioUrl": "https://api.itslanguage.nl/download/4everAndever"
 }
```

or as result:
```http
 HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
   "id": "speech_1",
   "created": "2014-01-28T21:25:10Z",
   "updated": "2014-01-28T21:25:10Z",
   "audioUrl": "https://api.itslanguage.nl/download/YsjdG37bUGseu8-bsJ",
   "userId": "24",
   "resultSet": {
      "id": "4everAndever",
      "combinedAudioUrl": "https://api.itslanguage.nl/download/4everAndever"
   }
}
```

Reply:
I think the embedded `resultSet` object makes more sense, simply adding the audio attribute to the recording implies that it belongs to the recording. Will make the change.